server/models/contests/getModeratorContests.js

```diff
@@ -0,0 +1,31 @@
+const { pool } = require('../../models/database')
+
+/**
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {Object} param0.query
+ * @return {Promise}
+ */
+
+function getModeratorContests({ username, query }) {
+  return new Promise((resolve, reject) => {
+    const limit = Number(query.limit)
+    let sqlQuery = `SELECT * FROM contests WHERE id IN (SELECT contest_id FROM contests_moderators WHERE moderator=?)`
```

Comment:
Here again.. use JOIN
```suggestion
    let sqlQuery = `SELECT * FROM contests c JOIN contests_moderators cm ON c.id=cm.contest_id WHERE cm.moderator=? `
```

Reply:
I guess, here JOINS would be much slower